docs/manual/en/introduction/How-to-create-VR-content.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8">
+	<base href="../../../" />
+	<script src="list.js"></script>
+	<script src="page.js"></script>
+	<link type="text/css" rel="stylesheet" href="page.css" />
+</head>
+
+<body>
+	<h1>[name]</h1>
+	<br />
+
+	<p>
+		This guide provides a brief overview of the basic components of a web-based VR application
+		made with three.js.
+	</p>
+
+	<h2>Workflow</h2>
+
+	<p>
+		First, you have to include [link:https://github.com/mrdoob/three.js/blob/master/examples/js/vr/WebVR.js WebVR.js]
+		into your project.
+	</p>
+
+	<code>
+			&lt;script src="/path/to/WebVR.js"&gt;&lt;/script&gt;
+	</code>
+
+	<p>
+		*WEBVR.createButton()* does two important things: It creates a button which indicates
+		VR compatibility. Besides, it initiates a VR session if the user activates the button. The only thing you have
+		to do is to add the following line of code to your app.
+	</p>
+
+	<code>
+document.body.appendChild( WEBVR.createButton( renderer ) );
+	</code>
+
+	<p>
+		Next, you have to tell your instance of *WebGLRenderer* to enable VR rendering.
+	</p>
+
+	<code>
+renderer.vr.enabled = true;
+	</code>
+
+	<p>
+		Finally, you have to adjust your animation loop since we can't use our well known
+		*window.requestAnimationFrame()* function. The minimal code looks like this:
+	</p>
+
+	<code>
+function animate() {
+
+	renderer.setAnimationLoop( render );
+
+}
```

Comment:
I think this assumes some understanding/context of when `animate` is invoked and what `render` contains... I’ve seen examples where `animate()` would be the function called on each frame. Maybe to remove any confusion:

```js
renderer.setAnimationLoop( function () {

  renderer.render( scene, camera );

} );
```

Reply:
How about clearly writing "before" and "after"?

    // before
    function render() {

        requestAnimationFrame( render );
        renderer.render( scene, camera );

    }

    render();

    // after
    renderer.setAnimationLoop( function () {

        renderer.render( scene, camera );

    } );

Reply:
I think @donmccurdy should be sufficient. In this way, the code in the guide keeps compact.

Reply:
So, do you mind if adding a link to doc instead?https://threejs.org/docs/#api/en/renderers/WebGLRenderer.setAnimationLoop

I think animation loop setting is only the part some users often be missing/miswriting for VR contents creation. Actually I've written wrongly many times like this. The app doesn't show off any warnings/errors, but runs weirdly, then I've been confused. The doc could help them notice.

    function render() {

        requestAnimationFrame( render );  // I've forgotten to remove this line
        renderer.render( scene, camera );

    }

    renderer.setAnimationLoop( render );

Reply:
Okay, link is added.